pygeoapi/api.py

```diff
@@ -3096,11 +3096,23 @@ def describe_processes(self, request: Union[APIRequest, Any],
 
         if processes_config:
             if process is not None:
-                relevant_processes = [(process, processes_config[process])]
+                relevant_processes = [process]
             else:
-                relevant_processes = processes_config.items()
+                try:
+                    limit = int(request.params.get('limit'))
```

Comment:
We should type check as well.  See pattern in https://github.com/geopython/pygeoapi/blob/master/pygeoapi/api.py#L1355-L1372